### PR TITLE
Field guide section on W UMa binaries

### PR DESCRIPTION
**Pull request by janvanroestel**
The initial version of the W Uma page for the fieldguide. Has a description, an example lightcurve, an HR diagram and golden set.

### CODE REVIEW

File: data/golden/w_uma.csv [hunk 1]
```diff
@@ -0,0 +1,114 @@
+ra,dec
+73.3173, 56.2109
```
Comment (dmitryduev) [inline data/golden/w_uma.csv:2]: Please remove white spaces after commas throughout the file.

File: doc/field_guide__w_uma.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+### Classification and numbers
+- Supertypes
+  - variable
+  - periodic
+  - binary
+  - eclipsing
+- Occurrence rate: very common, about 10<sup>5</sup> expected in ZTF data
+
+
```
Comment (dmitryduev) [inline doc/field_guide__w_uma.md:12]:
```suggestion
```

**Review by dmitryduev — CHANGES_REQUESTED** [review]
Thanks @janvanroestel and congrats on your first PR 🚀 ! Please share your experience so far :)

Several requests:
- Please provide a descriptive name of the PR and a detailed description of the PR contents -- you should assume that a newcomer to the repo will understand what's going on here.
- Link the issue you submitted to this PR (see the right side of this page)
- Please do `git pull upstream main` -- I just committed a fix for the black pre-commit hook bug that was preventing code formatting. That may make the tests on the CI pass (but not necessarily, at least one will still legitimately fail, I think). In general, you want the tests to pass before inviting reviewers.
- Remove all png files (`git rm --cached doc/data/*png`), they get generated automatically. 
- Looks like you didn't commit the changes in `config.defaults.yaml`
- Please always tag at least 2 reviewers unless it's a really small PR. I'd volunteer @doccosmos or @mcoughlin ;)

**Comment by dmitryduev** [comment]
@janvanroestel any progress on responding to the review?

File: doc/field_guide__w_uma.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+#### Other characteristics and selection methods
+- g-r = -0.1 -- 1.0. Short period systems are typically redder.
+- absolute magnitudes are similar to main-sequence stars
+
+![HR diagram of W Uma](data/hr__w_uma.png)
+
+### References and further reading:
+- Sterken & Jasschek: Light curves of variable stars
+- Marsh et al, MNRAS 465, 4678–4689, 2017
```
Comment (mcoughlin) [inline doc/field_guide__w_uma.md:34]: Maybe we should also cite the Tom Prince paper?

**Review by mcoughlin — APPROVED** [review]

**Comment by dmitryduev** [comment]
@janvanroestel do you want to incorporate anything from https://github.com/ZwickyTransientFacility/scope/issues/7#issuecomment-807508741 into you PR? Otherwise, LGTM, would be happy to merge if you're done.

**Review by dmitryduev — APPROVED** [review]
LGTM! LMK when it's good to go in.

File: doc/field_guide__w_uma.md (hunk 3 above)
Comment (janvanroestel) [reply doc/field_guide__w_uma.md:33]: Franklin Marsh was Tom's SURF student

Comment (mcoughlin) [reply doc/field_guide__w_uma.md:33]: Ah awesome! I thought of the other Marsh ;)

**Comment by dmitryduev** [comment]
@janvanroestel merge conflict, please resolve.

**Pull request merged**